process.sh

```diff
@@ -5,270 +5,58 @@ die() { set +v; echo "$*" 1>&2 ; exit 1; }
 
 if [ "$#" -ne 0 ]; then
     die 'Collects source data, processes it, and pushes to S3.
-No commandline arguments, but looks for two environment variables:
-  "NO_PUSH=true" will fetch and process data, but not push to S3.
-  "CI=true" will use fixtures, rather than fetching, and also will not push to S3.
-'
+    No commandline arguments, but looks for two environment variables:
+    "NO_PUSH=true" will fetch and process data, but not push to S3.
+    "CI=true" will use fixtures, rather than fetching, and also will not push to S3.
+    '
 fi
 
 main() {
-  process_linnarson_cells
-  process_linnarson_molecules
-  process_linnarson_images
-  process_giotto
-  process_mermaid
-
-  echo
-  echo 'input:'
-  ls -lh "$INPUT"/*
-
-  echo
-  echo 'output:'
-  ls -lh "$OUTPUT"/*
-
-  echo
-  echo 'AWS:'
-  if [[ "$CI" = 'true' ]] || [[ "$NO_PUSH" = 'true' ]]
-  then
-    echo 'CI: Skip push to AWS'
-  else
-    # Exclude the *HUGE* PNGs in the base directory:
-    # The tiles for S3 are in subdirectories;
-    # We keep the PNGs around because it takes a long time to generate them.
-    aws s3 cp --exclude "$OUTPUT/*.png" --recursive "$OUTPUT" s3://"$S3_TARGET"
-  fi
+    echo
+    echo 'input:'
+    ls -lh "$INPUT"/
+
+    echo
+    echo 'output:'
+    ls -lh "$OUTPUT"/
+
+    sh ./scripts/process_linnarsson.sh
+    sh ./scripts/process_dries.sh
+    sh ./scripts/process_wang.sh
```

Comment:
`sh` != `bash`, at least potentially. Any reason not to just execute these? Then the `#!` takes precedence...
```suggestion
    ./scripts/process_wang.sh
```